INN/umbrella-borderzine
# Custom pop-out sidebar for article pages

## Changes

This pull request makes the following changes:

- Created new `borderzine_shortcode_sidebar` and `borderzine_do_sidebar` helper functions (borrowed from [scaip](https://github.com/INN/super-cool-ad-inserter-plugin/blob/master/inc/scaip-shortcode.php)) to allow the ability to display selected sidebars through a shortcode.
- Added basic styling for pop up created by pop up maker plugin.

![Screen Shot 2019-09-18 at 12 02 27 PM](https://user-images.githubusercontent.com/18353636/65165467-dafd3200-da0c-11e9-9eb4-758b73e9475b.png)

## Why



For #4

## Testing/Questions

Features that this PR affects:

- Article pages


Questions that need to be answered before merging:

- [ ] Should we add any more "default" styles to this?

Steps to test this PR:

1. Install and activate [pop up maker](https://wordpress.org/plugins/popup-maker/) plugin
2. Create new pop-up and and set it to display on all posts with a slide animation to the right and position to the right
3. Add the new sidebar shortcode to the pop up content 
`[borderzine_sidebar sidebar="your_sidebar"]`
4. View a post and verify the popup works and the sidebar is displayed inside of it as expected
